src/test/Distributor.t.sol

```diff
@@ -0,0 +1,215 @@
+pragma solidity =0.8.11;
+
+import "ds-test/test.sol";
+import "forge-std/Vm.sol";
+
+import "src/Distributor.sol";
+import "src/test/__fixtures/MintableERC20.sol";
+
+contract DistributorTest is DSTest
+{
+    Vm private vm = Vm(HEVM_ADDRESS);
+
+    Distributor private distributor; 
+    MintableERC20 private incomingToken = new MintableERC20("Wrapped Vechain", "WVET");
+    MintableERC20 private tokenToRecover = new MintableERC20("Recover Me", "RME");
+    address private recipient1 = address(1);
+    address private recipient2 = address(2);
```

Comment:
I'm on the fence here but I've gotten into the habit of using `mVariable` for any member variable not on the API. So in effect that means `private mVar` and `internal mVar`. I think it makes it very clear in tests and functions what is what. However, it's not perfect because public member variables have no prefix so they kinda break the rule and reduce its value.

Let me know what you think, it would look like this:
```suggestion
    Distributor private mDistributor; 
    MintableERC20 private mIncomingToken = new MintableERC20("Wrapped Vechain", "WVET");
    MintableERC20 private mTokenToRecover = new MintableERC20("Recover Me", "RME");
    address private mRecipient1 = address(1);
    address private mRecipient2 = address(2);
```

Reply:
suggestion adopted